src/test/regress/expected/upgrade_post_11_after.out

```diff
@@ -0,0 +1,31 @@
+SET search_path = post_11_upgrade;
+-- make sure that we always (re)sync the metadata
+UPDATE pg_dist_node_metadata SET metadata=jsonb_set(metadata, '{partitioned_citus_table_exists_pre_11}', to_jsonb('true'::bool), true);
+SELECT citus_finalize_upgrade_to_citus11(enforce_version_check:=false);
+NOTICE:  Preparing all the existing partitioned table indexes
+NOTICE:  Preparing to sync the metadata to all nodes
+ citus_finalize_upgrade_to_citus11
+---------------------------------------------------------------------
+ t
+(1 row)
+
+-- tables are objects with Citus 11+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) FROM citus.pg_dist_object WHERE objid IN ('post_11_upgrade'::regnamespace, 'post_11_upgrade.part_table'::regclass, 'post_11_upgrade.sensors'::regclass, 'post_11_upgrade.func_in_transaction_def'::regproc, 'post_11_upgrade.partial_index_test_config'::regconfig, 'post_11_upgrade.my_type'::regtype, 'post_11_upgrade.german_phonebook_unpropagated'::regcollation) ORDER BY 1;
```

Comment:
`regcollation`  only exists on PG 13+, so couldn't add to the test. But works fine manually tested